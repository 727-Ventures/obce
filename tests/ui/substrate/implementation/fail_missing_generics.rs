pub struct ChainExtension;

#[obce::definition]
pub trait ChainExtensionDefinition {}

#[obce::implementation]
impl ChainExtensionDefinition for ExtensionContext {}

fn main() {}
