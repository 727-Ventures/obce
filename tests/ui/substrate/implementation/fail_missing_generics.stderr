error: `ExtensionContext` should have 5 generics as `<'a, E, T, Env, Extension>`
 --> tests/ui/substrate/implementation/fail_missing_generics.rs:7:35
  |
7 | impl ChainExtensionDefinition for ExtensionContext {}
  |                                   ^^^^^^^^^^^^^^^^
